interfaces/index.ts

```diff
@@ -1,6 +1,7 @@
 import { AppProps } from "next/app";
 
 export type ComboListProps = {
```

Comment:
```suggestion
export type ComboList= {
```

propsは引数っていう意味だから、なくても良さそう